plenum/server/node.py

```diff
@@ -2108,6 +2108,12 @@ def allLedgersCaughtUp(self):
             logger.info('{}{} caught up till {}'
                         .format(CATCH_UP_PREFIX, self, last_caught_up_3PC),
                         extra={'cli': True})
+        to_pop = []
```

Comment:
I would move it into a separate method: `def _clean_non_forwarded_ordered`

Reply:
done